### ISSUE TITLE
Layout broken on IE9

### ISSUE THREAD

**Issue by osma** [NatLibFi/Skosmos#187]
Originally reported by @catecara

## At which URL did you encounter the problem?
http://oek1.fao.org/skosmos/agrovoc/en/

## What steps will reproduce the problem?
1. Use Skosmos with IE9 on Windows 7

## What is the expected output? What do you see instead?

The layout is broken. See screenshot below

![skosmos-ie9](https://cloud.githubusercontent.com/assets/1132830/5855533/f4331ab2-a241-11e4-8a3b-ba45d7bc91d9.png)

**Comment by henriyli** [comment]
I wasn't able to reproduce this issue with IE9 and WIN7.

**Comment by catecara** [comment]
Yes, still holds for me. Here a new screen shot. Slightly different from the one before because this morning Fabrizio and I made some changes.. but in its essence it is still visible: everything has moved to the left of the screen.

**Comment by osma** [comment]
@catecara Which exact IE9 version are you using? What is the screen resolution? Can you try with a smaller browser window?

Are you sure you are not seeing old content from the browser cache? Try hitting Ctrl+F5 to force a reload.

**Comment by catecara** [comment]
This is v 9.0.8112.16421. Resolution 1600x1200. Nothing changes if window size is changed. Yes, sure refresh done.

**Comment by henriyli** [comment]
Does removing that 125% zoom level help?

**Comment by catecara** [comment]
nope.

**Comment by henriyli** [comment]
Might have something to do with document mode. I can now reproduce this in the same version of ie9 by manually setting it to quirks or IE7 standards. 

I wonder if we should try to force it to the latest like suggested here:
http://stackoverflow.com/questions/10975107/force-to-use-internet-explorer-ie9-standards-document-mode

**Comment by osma** [comment]
Very likely explanation. IE also defaults to "compatility mode" (which is a misnomer as it really isn't compatible with recent web standards) when in an intranet zone. This could explain why @catecara is seeing this by default, while we have difficulty reproducing it.

+1 for testing this:
```html
<!doctype html>
 <meta http-equiv="X-UA-Compatible" content="IE=Edge">
```

**Comment by henriyli** [comment]
Added the meta tag mentioned before in abfcdab3880311f16b9213d1a2e15d0307f8156d

**Comment by osma** [comment]
I was able to replicate this by manually adding the skosmos hostname to the list of intranet sites in IE9 settings. The layout breaks down exactly as in the screenshots. But with the meta tag fix mentioned above, this no longer happens even if the site is in the intranet zone.

This fix will be in Skosmos 0.7.2, to be released soon.
